test/fn/edit-contacts.spec.js

```diff
@@ -192,6 +194,191 @@ describe('edit-contacts', function() {
+
+    expect(db.callCount).to.equal(1);
+    expect(db.args[0][0]).to.deep.equal({
+      keys: [ '0ebca32d-c1b7-5522-94a3-97dd8b3df146' ],
+      include_docs: true
+    });  
+    expect(fs.readJson(`${editContactsPath}/${editedJsonDocs}/7ac33d1f-10d8-5198-b39d-9d61595292f6.doc.json`)).to.deep.equal(
+      {
+        type: 'person',
+        name: 'kelly adisa',
+        is_in_emnch: false,
+        rbf: false,
+        _id: '7ac33d1f-10d8-5198-b39d-9d61595292f6'
+      }
+    );
+    expect(fs.readJson(`${editContactsPath}/${editedJsonDocs}/0ebca32d-c1b7-5522-94a3-97dd8b3df146.doc.json`)).to.deep.equal(
+      {
+        type: 'person',
+        name: 'janie',
+        is_in_emnch: true,
+        rbf: false,
+        _id: '0ebca32d-c1b7-5522-94a3-97dd8b3df146',
+        _rev: '1-ccec0024d98011c6d33c223ba389b1da'
+      }
+    );
+    expect(fs.readJson(`${editContactsPath}/${editedJsonDocs}/09efb53f-9cd8-524c-9dfd-f62c242f1817.doc.json`)).to.deep.equal(
+      {
+        type: 'health_center',
+        name: 'carla olamide',
+        is_in_emnch: false,
+        rbf: true,
+        _id: '09efb53f-9cd8-524c-9dfd-f62c242f1817'
+      }
+    );
+  });
+
+  it('should prompt a user if they will ovewrite files in a directory if they have not passed the updateOfflineDocs flag', async () => {
+    const prompt = sinon
+      .stub(userPrompt, 'keyInSelect')
+      .returns(0);
+    sinon
+      .stub(environment, 'extraArgs')
+      .get(() => ['--columns=type', '--files=contact.type.csv', `--docDirectoryPath=${editedJsonDocs}`]);
+    await editContactsModule.execute();
+
+    expect(prompt.callCount).to.equal(2);
+  });
+
+  it('should not prompt a user again if they choose to ovewrite all files', async () => {
+    const prompt = sinon.stub(userPrompt, 'keyInSelect').returns(1);
+    sinon
+      .stub(environment, 'extraArgs')
+      .get(() => ['--columns=type', '--files=contact.type.csv', `--docDirectoryPath=${editedJsonDocs}`]);
+    await editContactsModule.execute();
+
+    expect(prompt.callCount).to.equal(1);
+  });
+
+  it('should throw an error and exit if a user chooses not to overwrite files', async () => {
+    sinon
+      .stub(userPrompt, 'keyInSelect')
+      .returns(2);
+    sinon
+      .stub(environment,'extraArgs')
+      .get(() => ['--columns=type', '--files=contact.type.csv', `--docDirectoryPath=${editedJsonDocs}`]);
+    try {
+      await editContactsModule.execute();
+      assert.fail('should throw an error a user chooses not to overwrite files');
+    } catch (err) {
+      expect(err.message).to.be.equal('User canceled the action.');
+    }
+  });
+
+  it('should not prompt a user if the force flag is passed', async () => {
+    const prompt = sinon
+      .stub(userPrompt, 'keyInSelect')
+      .returns(1);
+    sinon
+      .stub(environment, 'extraArgs')
+      .get(() => ['--columns=type', '--files=contact.type.csv', `--docDirectoryPath=${editedJsonDocs}`, '--force=true',]);
```

Comment:
When force is passed here. It's being parsed as part of `extraArgs`. Is the context of force different than force passed in the main args? Meaning it will and perform the actions without user prompt? If so I think it would be worth updating this to use the main args force instead of having two --force params.